src/components/ProjectSummary.js

```diff
@@ -40,6 +43,10 @@ const ProjectSummary = (props) => {
         currencyInformation: currencyInformation
     })
 
+    const redirectToClient = () => {
```

Comment:
the word "redirect" is confusing because it is a term for redirecting old web URLs to new ones

change this to `goToClientPage`